Source/Scene/Cesium3DTilesetTraversal.js

```diff
@@ -307,7 +307,7 @@ define([
 
         // Use parent's geometric error with child's box to see if the tile already meet the SSE
         var parent = tile.parent;
-        if (defined(parent) && (parent.refine === Cesium3DTileRefine.ADD) && getScreenSpaceError(tileset, parent.geometricError, tile, frameState) <= tileset._maximumScreenSpaceError) {
+        if (defined(parent) && !parent.hasTilesetContent && (parent.refine === Cesium3DTileRefine.ADD) && getScreenSpaceError(tileset, parent.geometricError, tile, frameState) <= tileset._maximumScreenSpaceError) {
```

Comment:
This line is getting a bit unwieldy. Can we account for this in `updateVisibility`? Or if not just pull out

`getScreenSpaceError(tileset, parent.geometricError, tile, frameState)` 
or 
`getScreenSpaceError(tileset, parent.geometricError, tile, frameState) <= tileset._maximumScreenSpaceError`

into its own var?